### ISSUE TITLE
Write Co-authored-by in commit messages for translation updates

### ISSUE THREAD

**Issue by az0** [bleachbit/bleachbit#296]
https://help.github.com/articles/creating-a-commit-with-multiple-authors/

**Comment by abitrolly** [comment]
Is there is a script already that produces these commits? https://github.com/bleachbit/bleachbit/commit/f8dc91f6bfb6a9b7a5bfa3a44ca0016cfe9dc50e

**Comment by az0** [comment]
@abitrolly 
Yes, it is here https://github.com/bleachbit/bleachbit-misc/blob/master/lp2git.py

**Comment by az0** [comment]
this is obsolete by weblate
